torch/_dynamo/pgo.py

```diff
@@ -559,6 +559,7 @@ def hit(ty: str) -> DefaultDict[CodeId, CodeState]:
             "string",
             lambda: render_code_state(_CODE_STATE),
         )
+        set_feature_use("pytorch/remote_cache:dynamo_pgo_version", True)
```

Comment:
I am a bit confused about where we should put the `set_feature_use` call. Why do we not just automatically set feature use when the JK is consulted and successfully returns True? That seems like it would give us a much more uniform picture about when we are hitting the JK and it is True and we are therefore enabling the feature. Perhaps your response is, well, I don't care about if the JK returned True, what I actually care is if we got a non-empty PGO state from the local/remote cache lookup meaning that we are in a regime where PGO has affected the semantics of this run. But in that case, why would you name the feature a JK? It's actually pretty non-obvious what exactly this feature flag means. Here, it seems you are only instrumenting a hit on get. But PGO has an effect even if you didn't hit; we have to upload the profiles! (And empirically, I am definitely interested in knowing if a job is uploading PGO profiles, because that was the smoking gun in, e.g., the GC discussion). But if you just name the feature the JK there's no way I can intuit that, I'd have to read the code and see where the feature use was set to understand this.

Reply:
So the thought between me and @masnesral was to record actual use of the features in this column. I'm happy to rename this (and the other set_feature_calls) to not use JK keys (and some of them already do).

Reply:
That is fine, just don't use JK then for the key